docs/content/docs/recipes/teams.md

```diff
@@ -0,0 +1,347 @@
+
+{{< /callout >}}
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project, e.g., `/My Microsoft Teams Projects`.
+1. Create a vanilla Teams tab application by running the below command. See [the Microsoft Teams documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/create-channel-group-tab?tabs=nodejs) for more information on setting up Teams applications.
+
+    ```dotnetcli
+    yo teams
+    ```
+
+1. After creating the project, navigate to it with the command `cd <your project name>`.
+1. The project will use three Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    | `fluid-framework`    |Contains the IFluidContainer and other [distributed data structure]({{< relref "dds.md" >}}) that synchronizes data across clients.|
+    | `@fluidframework/azure-client`   |Defines the starting schema for the [Fluid container][].|
+    | `@fluidframework/test-client-utils` |Defines the `InsecureTokenProvider` needed to create the connection to a Fluid service.|
+    {.table}
+
+    Run the following command to install the libraries.
+
+    ```dotnetcli
+    npm install @fluidframework/azure-client fluid-framework @fluidframework/test-client-utils
+    ```
+
+## Code the project
+
+1. Open the file `/src/client/<your tab name>` in your code editor. Create a new file named `Util.ts`. Then add the following import statements:
+
+    ```ts
+    //`Util.ts
+
+    import { IFluidContainer } from "fluid-framework";
+    import { AzureClient, AzureClientProps, LOCAL_MODE_TENANT_ID } from "@fluidframework/azure-client";
+    import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+    ```
+
+### Defining Fluid functions and parameters
+
+1. Since this app is intended to be used in the context of Microsoft Teams, having all Fluid related imports, initialization, and functions in one place makes it easier to use in the future. Add the following code below the import statements. The comments will define all the functions and constants needed to interact with the Fluid service and container.
+
+```ts
+// TODO 1: Define the parameter key(s).
+// TODO 2: Define container schema.
+// TODO 3: Define connectionConfig (AzureClientProps).
+// TODO 4: Create Azure client.
+// TODO 5: Define create container function.
+// TODO 6: Define get container function.
+```
+
+1. Replace `TODO 1:` with the following code. Note that the constant is being exported because it will be used to append to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time.
```

Comment:
```suggestion
1. Replace `TODO: 1` with the following code. Note that the constant is being exported because it will be appended to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time.
```
`TODO: 1` to be consistent with the rest of the doc (though maybe the other ones should be revised to the `TODO 1: format instead?)

Reply:
I think `TODO 1:` makes more sense so that 1 isn't confused with the rest of the instruction